packages/google-cloud-dialogflow-cx/google/cloud/dialogflowcx_v3beta1/__init__.py

```diff
@@ -144,7 +144,6 @@
 )
 from .types.example import (
     Action,
-    ActionParameter,
```

Comment:
This breaking change is in the v3beta1 API so there will be no major version bump